samples/csharp/getting-started/Ranking_PersonalizedSort/PersonalizedRanking.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
```

Comment:
All the elements in this ItemGroup shouldn't be necessary. This whole group can be deleted.

Reply:
oh, I see you have a `.csproj` nested under another `.csproj`. This isn't typical. Can they be in sibling folders instead?

---
In reply to: [298226619](https://github.com/dotnet/machinelearning-samples/pull/533#discussion_r298226619) [](ancestors = 298226619)

Reply:
Yes, that was my mistake when I was copying and pasting some things around while cleaning up my solution.  I have deleted the extra .csproj file.

Reply:
#resolved